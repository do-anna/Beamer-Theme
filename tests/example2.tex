\documentclass{beamer}

\usepackage{lipsum}
\usepackage{todonotes}

%todo: ovviamente, questo andrà cambiato.
\mode<presentation>
{
  \usetheme{Luebeck}      
  \usecolortheme{orchid} 
  \usefonttheme{structuresmallcapsserif}  
  \setbeamertemplate{navigation symbols}{}
  \setbeamertemplate{caption}[numbered]
} 

\usepackage[italian]{babel}
\usepackage[utf8]{inputenc}

\AtBeginSection[]{
  \begin{frame}
  \vfill
  \centering
  \begin{beamercolorbox}[sep=8pt,center,shadow=true,rounded=true]{title}
    \usebeamerfont{title}\insertsectionhead\par%
  \end{beamercolorbox}
  \vfill
  \end{frame}
}

\title{Un breve test}
\subtitle{esempio di applicazione del template}
\author{autori vari}
\institute{Università di Trento}
\date{30 febbraio 2042}

\begin{document}

\begin{frame}
  \titlepage
\end{frame}

\begin{frame}{Contenuti}
  \tableofcontents
\end{frame}

\section{Titolo sezione}

\begin{frame}{Questo è un frame}
    \begin{block}{Un blocco di testo importante}
        \lipsum[66]
    \end{block}
    \pause
    \begin{itemize}
      \item \textbf{un punto}: posso scrivere cose
      \pause
      \item \textbf{altro punto}: posso scriverle due volte
    \end{itemize}
\end{frame}

\section{Altro titolo sezione}
\subsection{Sottosezione}

\begin{frame}{}
    \textbf{Dal vangelo secondo Lipsum:} \\
    \pause
    \textit{\lipsum[75]} \\ 
    (Lipsum, 75)
\end{frame}

\subsection{Seconda sottosezione}

\begin{frame}{Formule}
    \[
    \mathcal L_{\mathcal T}(\vec{\lambda})
    = \sum_{(\mathbf{x},\mathbf{s})\in \mathcal T}
       \log P(\mathbf{s}\mid\mathbf{x}) - \sum_{i=1}^m
       \frac{\lambda_i^2}{2\sigma^2}
    \]
\end{frame}

\begin{frame}{Formule 2}
    \begin{align*}
        S(\omega) 
        &= \frac{\alpha g^2}{\omega^5} e^{[ -0.74\bigl\{\frac{\omega U_\omega         19.5}{g}\bigr\}^{\!-4}\,]} \\
        &= \frac{\alpha g^2}{\omega^5} \exp\Bigl[ -0.74\Bigl\{\frac{\omega       U_\omega 19.5}{g}\Bigr\}^{\!-4}\,\Bigr] 
    \end{align*}
\end{frame}


\begin{frame}{Il problema delle immagini}
    \missingfigure[figwidth=6cm]{immagine a caso}
\end{frame}

\begin{frame}{Conclusione}
    \begin{alertblock}{Una frase profonda e filosofica}
        \lipsum[66]
    \end{alertblock}
\end{frame}

\end{document}


