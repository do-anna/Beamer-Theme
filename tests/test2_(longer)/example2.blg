This is BibTeX, Version 0.99d (TeX Live 2019)
Capacity: max_strings=100000, hash_size=100000, hash_prime=85009
The top-level auxiliary file: example2.aux
The style file: biblatex.bst
Reallocated glb_str_ptr (elt_size=4) to 20 items from 10.
Reallocated global_strs (elt_size=20001) to 20 items from 10.
Reallocated glb_str_end (elt_size=4) to 20 items from 10.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated wiz_functions (elt_size=4) to 6000 items from 3000.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Reallocated singl_function (elt_size=4) to 100 items from 50.
Database file #1: example2-blx.bib
Database file #2: biblio.bib
Biblatex version: 3.11
Reallocated wiz_functions (elt_size=4) to 9000 items from 6000.
Reallocated singl_function (elt_size=4) to 100 items from 50.
You've used 4 entries,
            6304 wiz_defined-function locations,
            1210 strings with 9289 characters,
and the built_in function-call counts, 18452 in all, are:
= -- 1346
> -- 445
< -- 220
+ -- 368
- -- 96
* -- 930
:= -- 1515
add.period$ -- 0
call.type$ -- 4
change.case$ -- 182
chr.to.int$ -- 34
cite$ -- 6
duplicate$ -- 1788
empty$ -- 1667
format.name$ -- 146
if$ -- 4179
int.to.chr$ -- 0
int.to.str$ -- 11
missing$ -- 0
newline$ -- 100
num.names$ -- 286
pop$ -- 1012
preamble$ -- 1
purify$ -- 188
quote$ -- 0
skip$ -- 959
stack$ -- 0
substring$ -- 1922
swap$ -- 544
text.length$ -- 87
text.prefix$ -- 3
top$ -- 1
type$ -- 124
warning$ -- 0
while$ -- 191
width$ -- 0
write$ -- 97
