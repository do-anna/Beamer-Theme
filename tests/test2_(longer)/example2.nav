\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {1}{1}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {2}{2}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@sectionpages {1}{2}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {1}{2}}
\defcounter {refsection}{0}\relax 
\headcommand {\sectionentry {1}{Titolo sezione}{3}{Titolo sezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {3}{3}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {1}{0}{2}{4/6}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {4}{6}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@sectionpages {3}{6}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {3}{6}}
\defcounter {refsection}{0}\relax 
\headcommand {\sectionentry {2}{Altro titolo sezione}{7}{Altro titolo sezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{0}{1}{7/7}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {7}{7}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {7}{7}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionentry {0}{2}{1}{8}{Sottosezione}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{1}{1}{8/9}{Sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {8}{9}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {8}{9}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionentry {0}{2}{2}{10}{Seconda sottosezione}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{1}{10/10}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {10}{10}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{2}{11/11}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {11}{11}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{3}{12/12}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {12}{12}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{4}{13/13}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {13}{13}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{5}{14/15}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {14}{15}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{6}{16/16}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {16}{16}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{7}{17/17}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {17}{17}}
\headcommand {\beamer@partpages {1}{17}}
\headcommand {\beamer@subsectionpages {10}{17}}
\headcommand {\beamer@sectionpages {7}{17}}
\headcommand {\beamer@documentpages {17}}
\headcommand {\gdef \inserttotalframenumber {13}}
