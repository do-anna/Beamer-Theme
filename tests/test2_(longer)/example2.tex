\documentclass{beamer}

\usepackage{lipsum}
\usepackage{todonotes}

%todo: ovviamente, questo andrà cambiato
\usepackage{../../beamerthemebeunitn}


\usepackage[italian]{babel}
\usepackage[utf8]{inputenc}


\title{Un breve test}
\subtitle{esempio di applicazione del template}
\author{autori vari}
\institute{Università di Trento}
\date{30 febbraio 2042}

\begin{document}
% c'è un overfull hbox, ma tanto è solo un dummy test

\begin{frame}
  \titlepage
\end{frame}

\begin{frame}{Contenuti}
  \tableofcontents
\end{frame}

\section{Titolo sezione}

\begin{frame}{Questo è un frame}
    \begin{block}{Un blocco di testo importante}
        \lipsum[66]
    \end{block}
    \pause
    \begin{itemize}
      \item \textbf{un punto}: posso scrivere cose
      \pause
      \item \textbf{altro punto}: posso scriverle due volte
    \end{itemize}
\end{frame}

\section{Altro titolo sezione}
\subsection{Sottosezione}
\begin{frame}{Questo è un frame dal titolo decisamente lungo}
    \textbf{Dal vangelo secondo Lipsum:} \\
    \pause
    \textit{\lipsum[75]} \\ 
    (Lipsum, 75)
\end{frame}

\subsection{Seconda sottosezione}

\begin{frame}{Formule}
    \[
    \mathcal L_{\mathcal T}(\vec{\lambda})
    = \sum_{(\mathbf{x},\mathbf{s})\in \mathcal T}
       \log P(\mathbf{s}\mid\mathbf{x}) - \sum_{i=1}^m
       \frac{\lambda_i^2}{2\sigma^2}
    \]
\end{frame}

\begin{frame}{Formule 2}
    \begin{align*}
        S(\omega) 
        &= \frac{\alpha g^2}{\omega^5} e^{[ -0.74\bigl\{\frac{\omega U_\omega         19.5}{g}\bigr\}^{\!-4}\,]} \\
        &= \frac{\alpha g^2}{\omega^5} \exp\Bigl[ -0.74\Bigl\{\frac{\omega       U_\omega 19.5}{g}\Bigr\}^{\!-4}\,\Bigr] 
    \end{align*}
\end{frame}


\begin{frame}{Il problema delle immagini}
    \missingfigure[figwidth=6cm]{immagine a caso}
\end{frame}

\begin{frame}{Conclusione}
    \begin{alertblock}{Una frase profonda e filosofica}
        \lipsum[66]
    \end{alertblock}
    \beamerskipbutton{Bottone finto}
\end{frame}

\begin{frame}
	\tableofcontents
\end{frame}

\end{document}


