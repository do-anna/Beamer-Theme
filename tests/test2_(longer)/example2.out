\BOOKMARK [2][]{Outline0.1}{Titolo sezione}{}% 1
\BOOKMARK [2][]{Outline0.2}{Altro titolo sezione}{}% 2
\BOOKMARK [3][]{Outline0.2.1.8}{Sottosezione}{Outline0.2}% 3
\BOOKMARK [3][]{Outline0.2.2.10}{Seconda sottosezione}{Outline0.2}% 4
